openedx/features/pakx/lms/overrides/tasks.py

```diff
@@ -43,6 +43,7 @@ def _get_email_message_context(data):
     message_context = get_base_template_context(site)
     message_context.update({
         'platform_name': configuration_helpers.get_value('PLATFORM_NAME', settings.PLATFORM_NAME),
+        'full_name': (user.profile.name or user.username).title(),
```

Comment:
it should be `username` or `first_name`

Reply:
username always exists, so send option will never work. also there is a strong chance that first name will be empty since we mainly (open edx) use profile's name.